admin/controllers/service-manager.js

```diff
@@ -323,31 +251,18 @@ const controller = {
    * @returns {Promise.<*>}
    */
   removeCheckWindow: async (req, res, next) => {
-    let checkWindow
-
     if (!req.params.checkWindowId) {
       return res.redirect('/service-manager/check-windows')
     }
-
+    let result
```

Comment:
Can this not be const?

Reply:
I don't see anything wrong with declaring a reassignable variable right before the try/catch block.
It could change to `const` but it's an approach we are using across most of the controllers.

Reply:
Sorry, it's fine, the diff wasn't making that clear